rllab/envs/mujoco/gather/embedded_viewer.py

```diff
@@ -1,22 +1,26 @@
-from rllab.mujoco_py import glfw, mjcore
-import rllab.mujoco_py.mjconstants as C
-from rllab.mujoco_py.mjlib import mjlib
+from mujoco_py import MjRenderContext, MjSim, functions, const as C
+from mujoco_py.generated.const import CAT_ALL
```

Comment:
PEP8 imports

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
- "The easy solution is to `import mujoco_py` _before_ `import glfw`."
This comment is specified by the creators of mujoco_py; should I make a note of this fact using a comment?

See: https://github.com/openai/mujoco-py/blob/6ac6ac203a875ef35b1505827264cadccbfd9f05/mujoco_py/builder.py